packages/wrangler/src/pages/projects.tsx

```diff
@@ -44,8 +44,11 @@ export async function ListHandler() {
 		account_id: accountId,
 	});
 
-	render(<Table data={data}></Table>, { patchConsole: false });
-	metrics.sendMetricsEvent("list pages projects");
+	const { unmount } = render(<Table data={data}></Table>, {
+		patchConsole: false,
+	});
+	unmount();
```

Comment:
We need to unmount apps to release control of this output from Ink.js.
Otherwise any subsequent calls to Ink.js's `render()` will wipe it out.

This should be standard practice across our codebase.